### ISSUE TITLE
Chapter 9 and 10

### ISSUE THREAD

**Issue by thatMeow** [LeiG/Applied-Predictive-Modeling-with-Python#6]
Hi, 

Could you share chapter 9 and 10 as well? Or do you intentionally deleted them?

Thanks!

**Comment by LeiG** [comment]
Nope, I just never get the chance to it :( Free feel to add a patch if you get to the rest of the chapters 💪
